thermal_image_topic_name: image_raw
thermal_image_raw_topic_name: thermal_image_raw
colored_thermal_image_topic_name: image_colored
camera_frame_id: thermal_camera
camera_name: thermal_camera
camera_info_url: file://${ROS_HOME}/camera_info/${NAME}.yaml
interpolate_dead_pixels: true
calibrate_dead_pixels: false
mean_compensation: false
calibrate_mean_compensation: false
denoise: true
show_debug_images: false
